onadata/libs/serializers/project_serializer.py

```diff
@@ -168,12 +169,22 @@ class Meta:
             'num_of_submissions',
             'downloadable',
             'encrypted',
+            'published_by_formbuilder',
             'last_submission_time',
             'date_created',
             'url',
             'last_updated_at'
         )
 
+    def get_published_by_formbuilder(self, obj):
+        md = obj.metadata_set.filter(
+            data_type='published_by_formbuilder'
```

Comment:
You can use `unique_type_for_form` function from the metadata to retrieve the value. It has implemented this already